src/components/tutor-input.cjsx

```diff
@@ -17,18 +17,22 @@ TutorInput = React.createClass
   onChange: (event) ->
     @props.onChange(event.target?.value, event.target)
 
+  focus: ->
+    React.findDOMNode(@refs.input)?.focus()
```

Comment:
This adds a `focus` method to the `TutorInput` component, which is then called by `add-student-cjsx`  The `?` is used since it's possible that it might be called while the component is unmounted, in which case `findDOMNode` would return null.

The whole need for the explicit focus comes from them being rendered inside a popover component so they're invisible when first rendered.  That defeats the normal autofocus attribute on an input.